docker/Dockerfile.test

```diff
@@ -1,9 +1,32 @@
-FROM screenly/srly-ose-base:latest-x86
+FROM screenly/srly-ose-base:latest-x86 as builder
```

Comment:
So this is fine. You could use the `srly-ose-base:latest-x86` as the builder base, or just `debian:buster` with whatever extras you need to compile.

Reply:
Got it, thanks! I'm currently trying out `srly-ose-base:latest-x86` as the base image. Will try using `debian:buster` later.